ai/yolo/auto_labeling_script/main.py

```
@@ -0,0 +1,206 @@
+
+    return best_box
+
+
+def process_one_pair(full_image_path, cropped_image_path, output_txt_path,
+                     class_id=0, preview_path=None,
+                     min_size=40, max_size=320, step=4, min_score=0.45):
+    full_img = cv2.imread(full_image_path)
+    cropped_img = cv2.imread(cropped_image_path)
+
+    if full_img is None:
+        raise ValueError(f"Nie udało się wczytać obrazu: {full_image_path}")
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)